tests/integration/pilot/testdata/mcs-serviceexport-crd.yaml

```diff
@@ -0,0 +1,93 @@
+# Copyright 2020 The Kubernetes Authors.
```

Comment:
nit: comment where its from

Reply:
You mean github repository location? 

I assume this is the original: https://github.com/kubernetes-sigs/mcs-api/blob/master/config/crd/multicluster.x-k8s.io_serviceexports.yaml

@josephpeacock can you confirm?